### ISSUE TITLE
Design Comparison Review

### ISSUE THREAD

**Issue by T0nj3** [NoroffFEU/musikkforandrerliv.no#580]
Look over and compare the code/website with the design in Figma. (Hifi-FedJan24)

Comment below on what is different from the design. It can be small differences or big differences. (please include screenshots)

**Comment by T0nj3** [comment]
@Andlie94 This is exceptional!! Would you like another assignee as there is a lot of work and many pages to go through?

**Comment by Andlie94** [comment]
Yes, just make sure that they write here which page they are comparing. 
I'm working on about page now :)

**Comment by T0nj3** [comment]
Sure! I'll add Nicolai from design-team actually!

**Comment by Andlie94** [comment]
Great :) Thank you!

**Comment by nicolaibye** [comment]
Hey! I'll start tackling Our Work!

Just a few comments about the index page:
The "Just another hand" font is for the headings, this wasn't updated in the design we were provided but has been updated in the HiFi - FEDJAN24 frame

Also updated here is the pink color. We have decided to change it to orange and not use pink at all.

Other than this great job! ^^

**Comment by nicolaibye** [comment]
Here is the write up for the Our Work page:
[Our_Work.docx](https://github.com/user-attachments/files/17204063/Our_Work.docx)

**Comment by Andlie94** [comment]
[Index_upodate.pdf](https://github.com/user-attachments/files/17204077/Index_upodate.pdf)
[About page.pdf](https://github.com/user-attachments/files/17204079/About.page.pdf)

here is the about page and updated index page.

**Comment by nicolaibye** [comment]
Here is the write up for the news page:
[News.docx](https://github.com/user-attachments/files/17204367/News.docx)

**Comment by Andlie94** [comment]
Here is the gallery page
[Uploading Gallery page.pdf…]()

**Comment by nicolaibye** [comment]
Here is the news post page:
[News_post.docx](https://github.com/user-attachments/files/17208059/News_post.docx)

**Comment by Andlie94** [comment]
Event page
[Events.pdf](https://github.com/user-attachments/files/17209916/Events.pdf)

**Comment by nicolaibye** [comment]
Contact Us feedback:
[Contact_Us.docx](https://github.com/user-attachments/files/17228586/Contact_Us.docx)

**Comment by nicolaibye** [comment]
Terms and Conditions seem untouched as it only has the NAV bar and footer (NAV/Footer need to be updated based on the feedback provided in the Our_Work document), missing the content for Terms and Conditions. Heading, blurb and dropdown sections need to be implemented.

**Comment by nicolaibye** [comment]
404 only have the basics added (head, body, footer), nav is missing in the HTML and styling is not connected. The 404 page not found content with image also needs to be added.
![image](https://github.com/user-attachments/assets/97c3dff7-2fdc-4306-af59-ca35e0396d37)

**Comment by Andlie94** [comment]
[Privacy Policy.pdf](https://github.com/user-attachments/files/17228904/Privacy.Policy.pdf)
Privacy Policy

**Comment by T0nj3** [comment]
Every task has been made from this. Thank you @Andlie94  and @nicolaibye !!
